Добавил пакет уведомлений для таймера и расширил старый функционал

webDeveloperSergey opened the pull request.

webDeveloperSergey committed:
Добавил пакет уведомлений для таймера и расширил старый функционал

webDeveloperSergey committed:
маленький фикс

EsilSemitone commented on 7-timer-notification/index.js:
```diff
@@ -0,0 +1,40 @@
+const sentNot = require('./sentNot')
+
+const timeArr = process.argv.slice(2)
+
```
Лучше иметь какую-то функцию main в которой происходит вся логика (получение ввода, валидация, установка таймера)

EsilSemitone commented on 7-timer-notification/index.js:
```diff
@@ -0,0 +1,40 @@
+
+const timeArr = process.argv.slice(2)
+
+const getFullTimeSeconds = () => {
```
Лучше вынести в папку tools или common

webDeveloperSergey committed:
Рефактор таймера

вынес вспомогательные функции в common + обернул приложение в единую точку входа main()
